service/src/test/java/datawave/microservice/audit/replay/AuditReplayTest.java

```diff
@@ -190,22 +189,22 @@ public void singleAuditReplayIgnoreUnfinishedTest() throws Exception {
         } catch (Exception e) {
             exception = e;
         }
-        Assert.assertNotNull(exception);
-        Assert.assertTrue(exception instanceof HttpClientErrorException);
-        Assert.assertTrue(((HttpClientErrorException) exception).getResponseBodyAsString().contains("Cannot stop audit replay with id " + replayId));
+        Assertions.assertNotNull(exception);
```

Comment:
consider correcting these to be static imports

Reply:
same comment as above.